orc8r/cloud/go/mproto/test/mproto_test.go

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Note: don't put tests in `test/` directories, instead put them inline with the code they test

The reason we're making a `test/` directory is to hold the test protos ("test util")